v1/types/types.gunk

```diff
@@ -195,508 +195,31 @@ const (
 	Transferwise
 )
 
-// Currency following format specified on ISO 4217
```

Comment:
btw miggy, why const of currency is deleted ?

Reply:
@nkristianto the enum *CurrencyISO4217* is used in ob-services are referred from `transfer` package and not from types. 
@mgfcmartinez CMIIW